_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -813,7 +813,7 @@ VAAPIVideoCORE::GetVAService(
 } // mfxStatus VAAPIVideoCORE::GetVAService(...)
 
 mfxStatus
-VAAPIVideoCORE::SetCmCopyStatus(bool enable)
+VAAPIVideoCORE::SetCmCopy(bool enable)
```

Comment:
I believe no sense for this function to return status

Reply:
@mgonchar  Is it possible to change `mfxStatus CMEnabledCoreInterface::SetCmCopy(bool enable)` to return void as well? Can it affect ABI of some plugins?

Reply:
@onabiull : please, comment.

Reply:
If we wan't old plugins to remain functional, we need change interfaces by adding new ones. This would affect windows version in the first hand.

Reply:
@onabiull : Does this apply to VAAPIVideoCORE or VideoCORE or both?

Reply:
If I understand Oleg correctly, `CMEnabledCoreInterface` is part of plugin API and so change of return value's type is prohibited.

So, rename `virtual mfxStatus CMEnabledCoreInterface::SetCmCopyStatus(bool enable)` to `virtual mfxStatus CMEnabledCoreInterface::SetCmCopy(bool enable)` is also incorrect and must be reverted.